src/engraving/dom/score.cpp

```diff
@@ -3647,10 +3647,10 @@ void Score::selectRange(EngravingItem* e, staff_idx_t staffIdx)
         if (startSegment) {
             Segment* endSegment = findElementEndSegment(this, e, m_selection.endSegment());
             staff_idx_t elementStaffIdx = e->staffIdx();
-            Fraction tick = startSegment->tick();
-            Fraction etick = endSegment->tick();
-
             if (endSegment && elementStaffIdx != muse::nidx) {
+                Fraction tick = startSegment->tick();
+                Fraction etick = endSegment->tick();
```

Comment:
This change is unrelated, but I noticed that it's weird to use `endSegment` before doing a null check on it...